contributors/design-proposals/propagation.md

```diff
@@ -65,6 +65,12 @@ and references to network namespaces persist.
 The new `VolumeMount` will look like:
 
 ```go
+const (
+	MountPropagationRShared  MountPropagationMode = "RShared"
+	MountPropagationRSlave   MountPropagationMode = "RSlave"
```

Comment:
`MountPropagationRecursiveShared` sounds really ugly and it's not used nowhere in kernel / docker  documentation. Renamed to `Shared` with a note that it's applied recursively (imo it's still quite confusing...)